app/frontend/styles/base/_utilities.scss

```diff
@@ -72,6 +72,10 @@ b {
   font-weight: bold;
 }
 
+.text-capitalize {
+  text-transform: capitalize;
```

Comment:
Do we want to capitalize the first letter of every word in the phrase, or just the first letter of the phrase? (thinking of the search results heading here)

Reply:
im waiting to confirm 100% with chris. i know using this css doesnt match the figma 100% as figma suggests should be first word only capitalised

Reply:
Can you apply `uppercase` to `selector::first-letter`? I've never used it before

Reply:
agreed with chris - keyword in heading should be exactly as user types it, location should be capitalised whatever